lib/UserBackend.php

```diff
@@ -35,8 +35,8 @@
 use OCP\IUserBackend;
 use OCP\IUserManager;
 use OCP\User\Events\UserChangedEvent;
+use OCP\User\Events\UserFirstTimeLoggedInEvent;
```

Comment:
Breaking, since until now we support Nextcloud 25-27, but I guess 28 might be a good cut then to have another stable branch